src/build/__init__.py

```diff
@@ -161,6 +163,20 @@ def __init__(self, srcdir='.', config_settings=None, python_executable=sys.execu
             python_executable=python_executable,
         )
 
+    @property
+    def python_executable(self):  # type: () -> Union[bytes, Text]
+        print(self.hook.python_executable, type(self.hook.python_executable))
```

Comment:
I think you must've inserted the `print` here for debugging?

Reply:
Yes, just removed.